### PR TITLE
Create Mitiq and Stim blog post

### PR DESCRIPTION
**Pull request by Misty-W**
Create a blog post summarizing the newly released tutorial using Mitiq to apply ZNE on logical circuits simulated on the Stim backend.

### CODE REVIEW

**Comment by Misty-W** [comment]
Open item: how to handle the references- I didn't see a bib file like we have in the Mitiq docs.

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## About Stim
+Stim is a quantum stabilizer circuit simulation package, specialized for the simulation of quantum error correction (QEC) circuits.
+It can be used as a Python package, a command line tool, or a C++ library.
+More information can be found on the [Stim project page](https://github.com/quantumlib/Stim). 
+
+Although Mitiq does not support Stim directly as a frontend, it can interface with the Stim backend through the Cirq and Stimcirq packages. Stimcirq is a library for translating between Cirq and Stim.
+
+## A Mitiq + Stim tutorial for applying error mitigation on logical circuits
+
+![Mitiq and Stim logos](/images/mitiq_stim_logo.png)
```
Comment (nathanshammah) [inline src/content/blog/2023_mitiq_stim_workflow.md:28]: Very nice but would be cool to make the image smaller. It's up to you to upload the image filesystem directly or use Cloudinary, but there's a small benefit of Cloudinary over using the filesystem as Cloudinary provides some image optimisation. If you use the filesystem, I recommend reducing a bit the size. You could try embedding some html with size control on the image (or maybe add a bit of white padding to the image?).

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The idea for a tutorial with Stim was first inspired by a comment from a community member on the arXiv pre-print of Ref. [3], suggesting the use of deeper circuits on the numerical example demonstrating a novel noise scaling technique proposed in the paper.
```
Comment (nathanshammah) [inline src/content/blog/2023_mitiq_stim_workflow.md:30]: Maybe could mention that the comment on Scirate. Just an idea as it's also OSS!

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Mitiq is the leading quantum error mitigation compiler and has recently hit 100k downloads and is developed by a community of over 50 contributors worldwide.
+
+## About Stim
+Stim is a quantum stabilizer circuit simulation package, specialized for the simulation of quantum error correction (QEC) circuits.
+It can be used as a Python package, a command line tool, or a C++ library.
+More information can be found on the [Stim project page](https://github.com/quantumlib/Stim). 
+
+Although Mitiq does not support Stim directly as a frontend, it can interface with the Stim backend through the Cirq and Stimcirq packages. Stimcirq is a library for translating between Cirq and Stim.
+
+## A Mitiq + Stim tutorial for applying error mitigation on logical circuits
```
Comment (nathanshammah) [inline src/content/blog/2023_mitiq_stim_workflow.md:26]: The text paragraph is a bit long here. What about breaking it into two, with two titles, one focusing on the content and one speaking about open science and the genesis of it? Just a proposal.

**Review by nathanshammah — COMMENTED** [review]
Thanks @Misty-W! I left some comments on the text, broadly speaking to make the narrative a bit more general and the beginning of the blog even more approachable to non experts. Let's discuss which ones you agree upon, many are suggestions.

By the way references look great.

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+
+In Mitiq v0.29.0 we added a [tutorial demonstrating a method of combining quantum error mitigation (QEM) and quantum error correction (QEC)](https://mitiq.readthedocs.io/en/stable/examples/zne_logical_rb_cirq_stim.html), with the goal of reducing the effective logical error rate of the computation.
+While QEM techniques such as zero noise extrapolation (ZNE) and probabilistic error cancellation are typically thought of as belonging to the NISQ regime, recently it has been shown that they can also benefit applications within the fault-tolerant regime [^1][^2][^3].
+In this example ZNE is applied with noise scaling by global unitary folding on logical randomized benchmarking (RB) circuits, and the use of Mitiq with the Stim stabilizer simulator as the backend is introduced.
+
+## About Mitiq
+[Mitiq](https://mitiq.readthedocs.io/en/latest/) is a compiler that makes programs more robust to errors in quantum computers.
+Because quantum computers have high error rates, compiling for error robustness is critical for useful applications.
```
[the remark this answers is not in the document]
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md]: I added this info.

File: src/content/blog/2023_mitiq_stim_workflow.md (hunk 2 above)
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md]: Good point, I added a brief mention of Scirate

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Mitiq is the leading quantum error mitigation compiler and has recently hit 100k downloads and is developed by a community of over 50 contributors worldwide.
+
+## About Stim
+Stim is a quantum stabilizer circuit simulation package, specialized for the simulation of quantum error correction (QEC) circuits.
+It can be used as a Python package, a command line tool, or a C++ library.
+More information can be found on the [Stim project page](https://github.com/quantumlib/Stim). 
+
+Although Mitiq does not support Stim directly as a frontend, it can interface with the Stim backend through the Cirq and Stimcirq packages. Stimcirq is a library for translating between Cirq and Stim.
```
[the remark this answers is not in the document]
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md:24]: Makes sense, I've removed this part and refer to the tutorial instead.

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+title: ZNE on logical circuits with Mitiq and the Stim backend
```
[the remark this answers is not in the document]
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md:2]: Agree the more generic title is better.

File: src/content/blog/2023_mitiq_stim_workflow.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md:16]: Good point, added.

File: src/content/blog/2023_mitiq_stim_workflow.md (hunk 1 above)
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md:28]: I didn't see any options to adjust the image on Cloudinary- I guess you meant it does it automatically? I'm using the Cloudinary version now, and both previews look the same to me.

File: src/content/blog/2023_mitiq_stim_workflow.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+title: ZNE on logical circuits with Mitiq and the Stim backend 
+author: Misty Wahl
+day: 14
+month: 9
+year: 2023
+---
+
+
+In Mitiq v0.29.0 we added a [tutorial demonstrating a method of combining quantum error mitigation (QEM) and quantum error correction (QEC)](https://mitiq.readthedocs.io/en/stable/examples/zne_logical_rb_cirq_stim.html), with the goal of reducing the effective logical error rate of the computation.
```
[the remark this answers is not in the document]
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md]: I see that now. I removed the phrase about logical error rate and changed the phrasing of the subsequent sentences.

File: src/content/blog/2023_mitiq_stim_workflow.md (hunk 3 above)
Comment (Misty-W) [reply src/content/blog/2023_mitiq_stim_workflow.md:27]: Split it into two sections, "A Mitiq + Stim tutorial for applying error mitigation on logical circuits" and "Quantum community impact"

File: src/content/blog/2023_mitiq_stim_workflow.md (hunk 1 above)
Comment (nathanshammah) [reply src/content/blog/2023_mitiq_stim_workflow.md]: Sounds fine to me, @Misty-W!

**Review by nathanshammah — APPROVED** [review]
@Misty-W I don't know if your changes are done already, but for me it's already green light, as the blog post reads much better! When it is ok for you, feel free to merge it, so that it goes live without further wait time for review.

**Pull request merged**